File: csharp/examples/GetSignedForms.cs
using System;
namespace Examples.examples
{
    // Example of GET /signed-forms
    public class GetSignedForms : BaseRequest
    {
        private string _filingMethodId;
        private string _websiteId;

        public GetSignedForms()
        {
            dotenv.net.DotEnv.Load();
            _filingMethodId = Environment.GetEnvironmentVariable("FILING_METHOD_ID");
            _websiteId = Environment.GetEnvironmentVariable("WEBSITE_ID");

            Console.WriteLine($"GetSignedForms: _filingMethodIde={_filingMethodId} _websiteId={_websiteId}");
        }


        public override void SendRequest()
        {
            GetRequest($"signed-forms?filing_method_id={_filingMethodId}&website_id={_websiteId}");
        }
    }
    
}

File: php/examples/get_signed_forms.php
<?php
require_once 'base_request.php';

// Example of GET /signed-forms

$debug = true;
$filing_method_id   = $_ENV['FILING_METHOD_ID'];
$website_id   = $_ENV['WEBSITE_ID'];

$request_params = [
	'filing_method_id' => $filing_method_id,
	'website_id' => $website_id
];

send_request('GET', '/signed-forms', $request_params, '');